samply/Cargo.toml

```diff
@@ -11,6 +11,7 @@ readme = "README.md"
 
 [dependencies]
 
+samply-symbols = { version = "0.22.0", path = "../samply-symbols" }
```

Comment:
This isn't needed, you can just `use wholesym::samply_symbols` in the code - it won't make a difference but I think it's good practice to use the re-export in order to avoid mixing versions.